examples/with-cookie-auth/README.md

```diff
@@ -22,31 +22,12 @@ cd with-cookie-auth
 
 ### Run locally
 
-The repository is setup as a [monorepo](https://zeit.co/examples/monorepo/) so you can run start the development server with `now dev` inside the project folder.
-
-Install the packages of `/api` and `/www` using `npm` or `yarn`:
-
-```bash
-cd api
-npm install
-cd ../www
-npm install
-```
-
-Now you can start the development server in the root folder:
-
-```bash
-now dev
-```
-
-You can configure the `API_URL` environment variable (defaults to `http://localhost:3000`) with [Now env](https://zeit.co/docs/v2/development/environment-variables/#using-now.json) in the `now.json` file:
+After you clone the repository you can install the dependencies, run `now dev` and start hacking! You'll be able to see the application running locally as if it were deployed.
```

Comment:
Shouldn't this recommend `yarn dev`? Users may not have `now` installed.

Reply:
You are right. I will change it to `yarn dev`.